compass/machines/default.cfg

```diff
@@ -7,6 +6,3 @@ system = single_node
 
 # whether to use mpirun or srun to run the model
 parallel_executable = mpirun
-
-# cores per node on the machine
-cores_per_node = 4
```

Comment:
Was this deletion intentional?

Reply:
Yes, I want the default to be to detect the number of cores on a single-node machine, rather than assign it.  But this could be a different PR.

Reply:
This is the commit that changes that behavior:
https://github.com/xylar/compass/commit/7a2b638f592221bc96ce45577943a81af546320a

Reply:
I took this out and will make a separate PR.